packages/react/src/internal/indicator-container/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './indicator-container';
```

Comment:
Personally instead of adding an index, I would include the utility in the `indicator-container.tsx`, and export it from there.

This is because the only way you'd want to use this is when using an internal component itself.

At the end, they both would still be exported from `utils`.

It's up to you to decide which way to leave here.

Reply:
In general I like single export files, makes testing easier and better scoped, and maintainability simpler.
In special separating `.tsx` from `.ts` files is extra helpful because the parser doesn't get confused by `<T>` types.

Lastly, I just realised `splitContainerProps.ts` is missing unit tests, our `jest` config ignores everything inside `internal`.
I'll fix that, that'll probably make my arguments clearer.

Don't mind later removing `internal/indicator-container/index.ts` in favour of `internal/index.ts` though.

Reply:
Yes, indeed configuration now ignores all `components/**.*`.

What I meant is:

```
internal/indicator-container/indicator-container.tsx
internal/indicator-container/splitContainerProps.ts
```

Then `indicator-container.tsx`:

```ts
import { splitContainerProps } from './splitContainerProps.ts'

const IndicatorContainer = ...

export {
  IndicatorContainer,
  splitContainerProps,
};
```

Reply:
I see, that's a bit weird but I'll give it a try.